tests/markdown/mustache-syntax.test.js

```diff
@@ -291,13 +317,13 @@ describe('Injection: When an injection tag follows an element', ()=>{
 			expect(rendered, `Input:\n${source}`, { showPrefix: false }).toBe('<div class="block" style="color:red;"><p>text</p></div>');
 		});
 
-		it.failing('renders a div "text" with two injected styles', function() {
+		it.failing('renders a div "text" with two injected styles,included quotes', function() {

```

Comment:
Rather than overwriting this test, which was meant to test two normal CSS properties, can you please create a separate test for your string variable?